### PR TITLE
Bump tinycolor2 to 1.4.1 - as well as few dev-dependencies

### PR DESCRIPTION
**Pull request by archmoj**
@plotly/plotly_js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
     "strongly-connected-components": "^1.0.1",
     "superscript-text": "^1.0.0",
     "svg-path-sdf": "^1.1.3",
-    "tinycolor2": "^1.3.0",
+    "tinycolor2": "^1.4.1",
```
Comment (etpinard) [inline package.json]: Weird. Why isn't there any diff in the `package-lock.json` file for `tinycolor2`?

Comment (archmoj) [reply package.json]: It was resolved in lock right from the beginning!
<pre>
git blame -L 11245,+2 -- package-lock.json
671861a2da (etienne 2018-02-01 16:31:41 -0500 11245)     "tinycolor2": {
671861a2da (etienne <b>2018-02-01</b> 16:31:41 -0500 11246)       "version": "1.4.1",
</pre>
https://github.com/plotly/plotly.js/commit/671861a2dad8b2aff45961b3e6515012a4fdc99c#diff-32607347f8126e6534ebc7ebaec4853d

Comment (etpinard) [reply package.json]: Got it. `tinycolor2` was already listed as `1.4.1` in our package-lock, but we never propagate the change to the package.json.

Thanks very much for cleaning that up!

**Comment by etpinard** [comment]
:dancer: - don't worry about the failing License test, those shouldn't run on feature branch (for now :smirk: )

**Pull request merged**